packages/plugin-devtools/src/components/CommandsComponent.ts

```diff
@@ -97,4 +98,27 @@ export class CommandsComponent extends OwlComponent<CommandsProps> {
     selectCommand(commandIdentifier: string): void {
         this.state.selectedCommandIdentifier = commandIdentifier;
     }
+    /**
+     * Return the configured commands matching the given command identifier.
+     *
+     * @param commandIdentifier
+     */
+    matchingShortcuts(commandIdentifier: string): ConfiguredCommand[] {
+        return this.env.editor.plugins
+            .get(Keymap)
+            .mappings.flat()
+            .map(shortcut => shortcut.configuredCommand)
+            .filter(shortcut => shortcut.commandId === commandIdentifier);
```

Comment:
I don't like the "chained transformation on each line" pattern. It's in the devtools so I don't really care, but please don't use it in actual editor code.

For the sake of comparison, my original suggestion was (modified to return `ConfiguredCommand` instead of `Mapping`):
```typescript
const shortcuts = this.editor.plugins.get(KeyMap).mappings.flat();
const commands = shortcuts.map(shortcut => shortcut.configuredCommand);
return commands.filter(command => command.commandId === commandIdentifier);
```

Not sure which one is neater though. Yours looks simpler and more chaotic at the same time, it's very weird. Maybe I'm just not used to see this pattern ? What do you think ?

Reply:
I think not using the chaining pattern allows you to name variables which makes things clearer, but that at the same time it makes the code bulky, which can make it somewhat harder to read. So for me it's on a case by case basis. And I figured in this case that I liked it better like this because the names I could come up with didn't provide much added value.